apps/web/src/pages/get-started/components/CreateWorkflowButton.tsx

```diff
@@ -0,0 +1,73 @@
+import { useQuery } from '@tanstack/react-query';
+import styled from '@emotion/styled';
+
+import { colors } from '@novu/design-system';
+
+import { ROUTES } from '../../../constants/routes.enum';
+import { useCreateTemplateFromBlueprint } from '../../../api/hooks';
+import { errorMessage } from '../../../utils/notifications';
+import { getBlueprintTemplateById } from '../../../api/notification-templates';
+import { getWorkflowBlueprintDetails } from '../../../utils';
+import { TemplateCreationSourceEnum } from '../../templates/shared';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+
+export function CreateWorkflowButton({
+  blueprintIdentifier,
+  children,
+}: {
+  blueprintIdentifier: string;
+  children: string;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** Please rename this Prop or use `React` types for children‏